packages/core/utils/src/modules-sdk/abstract-module-service-factory.ts

```diff
@@ -466,9 +466,11 @@ export function abstractModuleServiceFactory<
           await this.eventBusModuleService_?.emit(
             primaryKeyValues_.map((primaryKeyValue) => ({
               eventName: `${kebabCase(model.name)}.deleted`,
-              data: isString(primaryKeyValue)
-                ? { id: primaryKeyValue }
-                : primaryKeyValue,
+              body: {
```

Comment:
Why not have `data` one level higher as before? It feels like unnecessary nesting IMO, since `data` already implies `body` to me.